acados/ocp_nlp/ocp_nlp_cost_external.c

```diff
@@ -217,6 +219,15 @@ int ocp_nlp_cost_external_model_set(void *config_, void *dims_, void *model_,
     {
         model->ext_cost_fun_jac_hess = (external_function_generic *) value_;
     }
+    else if (!strcmp(field, "ext_cost_fun_jac"))
+    {
+        model->ext_cost_fun_jac = (external_function_generic *) value_;
+    }
+    else if (!strcmp(field, "cost_custom_hess"))
```

Comment:
maybe `ext_cost_custom_hess` is more in line with the other namings

Reply:
or even maybe just 'ext_cost_hess'

Reply:
I think I renamed all the fields as we discussed.

Reply:
yes that was before our discussion